.github/workflows/generate-readme.yaml

```diff
@@ -0,0 +1,36 @@
+name: Autogenerate README
+
+on:
+  pull_request:
+    paths:
```

Comment:
I think this is limiting the action to only run when these files have changed, right? But we want to check that the readme is up-to-date whenever content is added, so we should probably remove these.